src/Error/ExceptionRenderer.php

```diff
@@ -184,7 +184,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
@markstory With my approach you always get a response, the downsides are more theoretical and any conflicts very unlikely and only relevant for debug mode anway :)